priv/repo/migrations/20200710132150_create_votes.exs

```diff
@@ -0,0 +1,27 @@
+defmodule LiquidVoting.Repo.Migrations.CreateVotes do
+  use Ecto.Migration
+
+  def change do
+    create table(:votes, primary_key: false) do
+      add :uuid, :uuid, primary_key: true
+      add :yes, :boolean, default: false, null: false
+      add :weight, :integer, default: 1
+
+      add :participant_uuid,
+          references(:participants, column: :uuid, type: :uuid, on_delete: :nothing)
+
+      add :proposal_url, :text, default: false, null: false
+      add :organization_uuid, :uuid
+
+      timestamps()
+    end
+
+    create index(:votes, [:participant_uuid])
```

Comment:
this index is outdated, given all queries will include :organization_uuid now - so we might as well make it a composite index in this PR, to keep with the clean slate spirit

Reply:
:participant_uuid + :organization_uuid?

Reply:
exactly 👍